catalogue/webapp/pages/api/items.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'isomorphic-unfetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ItemsWork, CatalogueApiError } from '@weco/common/model/catalogue';
+import { catalogueApiError } from '../../services/catalogue/common';
+
+function hasOwnProperty<
+  X extends Record<string, unknown>,
+  Y extends PropertyKey
+>(obj: X, prop: Y): obj is X & Record<Y, unknown> {
+  return obj.hasOwnProperty(prop);
+}
+
+export function isCatalogueApiError(
+  response: ItemsWork | CatalogueApiError
+): response is CatalogueApiError {
+  return Boolean(hasOwnProperty(response, 'type') && response.type === 'Error');
+}
+
+async function fetchWorkItems(
+  id: string
+): Promise<ItemsWork | CatalogueApiError> {
+  try {
+    const items = await fetch(
+      `https://api.wellcomecollection.org/catalogue/v2/works/${id}/items`,
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          'x-api-key': process.env.items_api_key || '',
+        },
+      }
+    ).then(resp => resp.json());
+    return items;
+  } catch (error) {
+    return catalogueApiError();
+  }
+}
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const { id } = req.query;
+  const workId = Array.isArray(id) ? id[0] : id;
+  const response = await fetchWorkItems(workId);
+  res.setHeader('Content-Type', 'application/json');
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  if (isCatalogueApiError(response)) {
+    res.status(response.httpStatus);
+  } else {
+    res.status(200);
+  }
+  res.json(response);
+};
```

Comment:
It is very important to think about where the URL of this page is going to be accessible given how we do routing of the app.

Currently this will be `/api/items` which isn't routed to the catalogue app.

I would decide that the route `/api/{webapp}` i.e. `/api/catalogue` is routed to this app, and `/api/content` is routed to that app, but it would mean that we need to put this a folder deeper.